README.md

```diff
@@ -1,13 +1,5 @@
 # Google Student Training in Engineering Program
+This repo contains Javier Domene's portfolio and STEP projects.
 
-This repo contains the projects you'll work on for the first half of STEP.
-This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
-
-To get started:
-
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/googleinterns/step.git`
-- Open the GitHub setup walkthrough: `teachme ~/step/walkthroughs/week-1-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+## Portfolio deployment link
+https://domene-step-2020.ue.r.appspot.com/
```

Comment:
Nice! I like the gifs :)

Reply:
Thanks, they are really old projects which I felt would fit in here